docs/user/azure/dnszone.md

```diff
@@ -0,0 +1,31 @@
+# Public DNS Zone
+
+Azure's DNS Zone service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup
+for the cluster to the outside world. To use OpenShift, you must have created a public DNS zone in Azure in
+the same subscription as your OpenShift cluster. You must also ensure the zone is "authoritative" for the domain. There are
+two ways to do this outlined below: root domain and subdomain. A root domain is `openshiftcorp.com`. A subdomain is of
```

Comment:
There doesn't seem to be a need to distinguish between root and subdomains in the following docs.  Maybe we can drop the paragraph content after `...  "authoritative" for the domain.`?

Reply:
It's fine it just  adds more information. let's keep it, as we have it for AWS too.